src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,40 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+import java.io.IOException;
+
+/**
+ * This class represents the command of marking a task as done
+ */
+public class DoneCommand extends Command {
+    private int i;
```

Comment:
Maybe you could look at `DeleteCommand.java`? It is the same issue. 😀 You could also check for subsequent classes too (i.e. `Parser.java`).